ptclient/ldap.c

```diff
@@ -1489,21 +1489,17 @@ static int ptsmodule_make_authstate_group(
                 }
             }
         }
-    } else {
-        rc = ptsmodule_expand_tokens(ptsm->group_base, canon_id, NULL, &base);
-        if (rc != PTSM_OK)
-            goto done;
     }
 
-    syslog(LOG_DEBUG, "(groups) about to search %s for %s", base, filter);
-
-
+    /* XXX i guess canon_id+6 skips over leading "group:" */
     rc = ptsmodule_expand_tokens(ptsm->group_base, canon_id+6, NULL, &base);
```

Comment:
I think the commit message should reflect that its this function that got called twice. I wasn't clear to me at the start of review, *which* call is duplicated.